pkg/obc/provisioner.go

```diff
@@ -243,6 +243,66 @@ func (p *Provisioner) Update(ob *nbv1.ObjectBucket) error {
 	return nil
 }
 
+// UpdateBucketTagging is not handled by lib-bucket-provisioner and is a customized function to update bucket tagging with OBC labels
+func UpdateBucketTagging(sysClient *system.Client, obc *nbv1.ObjectBucketClaim) error {
+	if obc == nil || obc.Labels == nil {
+		return fmt.Errorf("OBC is not provided or doesn't contain any label")
+	}
+
+	var err error
+	if sysClient == nil {
+		sysClient, err = system.Connect(false)
```

Comment:
I would take it this out from the function to the EventHandler and not support sysClient to be nil

Reply:
Done, Thanks